reference-implementation/test/readable-stream.js

```diff
@@ -122,26 +122,6 @@ test('ReadableStream reading a stream makes ready and closed return a promise fu
   );
 });
 
-test('ReadableStream avoids waiting-to-waiting transitions when immediately reading', t => {
```

Comment:
Yeah. We were determining the right `rs.ready` value to expose to the user looking at `rs` by looking at it in a fulfillment callback (which is run asynchronously to actual operation such as `reader.read()`. So, we could check the value while the stream is locked). This property is kinda side-effect of that approach, I guess.